src/main/java/pl/kurs/watercontainers/models/WaterContainer.java

```diff
@@ -0,0 +1,71 @@
+package pl.kurs.watercontainers.models;
+
+import pl.kurs.watercontainers.exceptions.InvalidCapacityException;
+import pl.kurs.watercontainers.exceptions.InvalidLevelException;
+
+import java.io.InvalidClassException;
+import java.io.Serializable;
+import java.util.Objects;
+
+
+public class WaterContainer implements Serializable {
+    static final long serialVersionUID = 1L;
+
+    private final String name;
+    private final double maxCapacity;
+    private double waterLevel;
+
```

Comment:
Please make this class java beans compatibile (visible default constructor ect.)

Reply:
all right then